database/intializeDB.go

```diff
@@ -11,38 +13,77 @@ import (
 )
 
 type config struct {
```

Comment:
Add connectionTimeout(After how much time an unsuccessful connection should time out) and idleConnectionTimecout(after how much time an idle connection which is not making any requests should timeout)